torchgen/gen.py

```diff
@@ -1955,11 +1980,20 @@ def gen_per_operator_headers(
         ("NativeMetaFunctions", "_meta"),
         ("NativeFunctions", "_native"),
     ]:
+        if (whitelist_keys is not None) and (len(whitelist_keys) == 1):
+            dispatch_namespace = list(whitelist_keys)[0].lower()
+        #     includes = f"#include <ATen/{dispatch_namespace}/ops/{name}{suffix}.h>"
+        # else:
+        #     includes = f"#include <ATen/ops/{name}{suffix}.h>"
```

Comment:
if not needed, just remove it.

Reply:
has removed, thanks for point out the issue